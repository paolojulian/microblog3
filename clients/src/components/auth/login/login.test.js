import React from 'react';
import { shallow } from 'enzyme';
import ErrorMsg from '../../widgets/form/error';

describe("Login", () => {
    it('renders form error message', () => {
        const errorMessage = 'Username or Password is incorrect';
        const component = shallow(<ErrorMsg error={errorMessage}/>);
        expect(component).toMatchSnapshot();
    });
});