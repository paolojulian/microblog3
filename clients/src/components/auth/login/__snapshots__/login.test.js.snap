// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login renders form error message 1`] = `ShallowWrapper {}`;
